src/test/java/ua/rent/masters/easystay/service/impl/AccommodationServiceImplTest.java

```diff
@@ -0,0 +1,176 @@
+package ua.rent.masters.easystay.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import ua.rent.masters.easystay.dto.accommodation.AccommodationRequestDto;
+import ua.rent.masters.easystay.dto.accommodation.AccommodationResponseDto;
+import ua.rent.masters.easystay.exception.EntityNotFoundException;
+import ua.rent.masters.easystay.mapper.AccommodationMapper;
+import ua.rent.masters.easystay.model.Accommodation;
+import ua.rent.masters.easystay.model.Amenity;
+import ua.rent.masters.easystay.repository.AccommodationRepository;
+import ua.rent.masters.easystay.repository.AmenityRepository;
+import ua.rent.masters.easystay.utils.TestDataUtils;
+
+@ExtendWith(MockitoExtension.class)
+class AccommodationServiceImplTest {
+
+    @InjectMocks
+    private AccommodationServiceImpl accommodationService;
+
+    @Mock
+    private AccommodationMapper accommodationMapper;
+
+    @Mock
+    private AmenityRepository amenityRepository;
+
+    @Mock
+    private AccommodationRepository accommodationRepository;
+
+    @Test
+    @DisplayName("Verify save() method works")
+    void save_ValidAccommodationRequestDto_ReturnsAccommodationResponseDto() {
+        // Given
+        AccommodationRequestDto requestDto = TestDataUtils.createAccommodationRequestDto();
```

Comment:
You can make it static import.
```suggestion
        AccommodationRequestDto requestDto = createAccommodationRequestDto();
```

Reply:
done